src/dmd/frontend.h

```diff
@@ -6609,6 +6609,7 @@ class StructLiteralExp final : public Expression
     Expression* getField(Type* type, uint32_t offset);
     int32_t getFieldIndex(Type* type, uint32_t offset);
     Expression* addDtorHook(Scope* sc);
+    Expression* toLvalue(Scope* sc, Expression* e);
```

Comment:
Also add this to expression.h and it can be merged.

Reply:
done